README.md

```diff
@@ -140,11 +142,16 @@ permissions have been set to allow it.
 }
 ```
 
-You can then run `chalice deploy --no-autogen-policy`.
+You can then run `chalice deploy --staging commonsstaging --no-autogen-policy`.
```

Comment:
In the last pull request, David asked why I added the `staging` option.

Since the Clean up Documentation Story is in the Staging Deployment Epic, I assumed that the readme is only for deploying staging instances.

Reply:
Yeah, I find it a bit confusing to mix the documentation for just getting started developing with setting up release environments. It's fine as long as it works, but I'd like to give developers as close to a "vanilla" chalice experience as possible to start?

Reply:
As @david4096 says, our readme files must at least include documentation on the simple "vanilla" deployment, using the minimum required set of options.
I also think it is feasible and important to include information on the typical/common variation(s) to the deployment as well, which for Lambda's is primarily the stage name.

What about something like this:
```
You can then run:
`chalice deploy --no-autogen-policy`.

Or, deploy to a specific AWS API Gateway stage, run:
`chalice deploy --no-autogen-policy --staging <stagename>`
```